DataSetDocker/Dockerfile

```diff
@@ -62,12 +61,38 @@ RUN conda install --quiet --yes "tornado<5.0.0"
 RUN conda install --quiet --yes pip
 RUN pip install py2nb
 
-RUN find /tmp/dataset/python/examples -name '*.py' -exec py2nb {} \; ;  \
-    bash -c "if [ ! -d /home/$NB_USER/work/examples ]; then mkdir /home/$NB_USER/work/examples; fi;" ; \ 
-    bash -c "if [ -d /tmp/dataset/python/examples ] && [ -d /home/$NB_USER/work/examples ]; then find /tmp/dataset/python/examples -name '*.ipynb' -exec mv {} /home/$NB_USER/work/examples \; ; fi;" ; \
-    chmod -R a+rwx /home/$NB_USER/work
+ARG UPD_EX=/usr/local/bin/update_examples.sh
+
+RUN touch $UPD_EX && \
+    chmod a+x $UPD_EX &&\
+    echo "#!/bin/bash" >> $UPD_EX && \
+    echo "cd /tmp/dataset && git checkout master && git pull" >> $UPD_EX && \
+    echo "find /tmp/dataset/python/examples -name '*.py' -exec py2nb {} \; ;" >> $UPD_EX && \
+    echo "if [ ! -d /home/$NB_USER/work/examples ]; then mkdir /home/$NB_USER/work/examples; fi;" >> $UPD_EX && \
+    echo "if [ -d /tmp/dataset/python/examples ] && [ -d /home/$NB_USER/work/examples ]; then find /tmp/dataset/python/examples -name '*.ipynb' -exec mv {} /home/$NB_USER/work/examples \; ; fi;" >> $UPD_EX && \
+    echo "chmod -R a+rwx /home/$NB_USER/work" >> $UPD_EX && \
+    echo "chmod -R a+rwx /tmp/dataset/python/examples" >> $UPD_EX && \
+    echo "chmod -R a+rwx /tmp/dataset/.git" >> $UPD_EX && \
+    echo "chown -R $NB_USER /home/$NB_USER/work" >> $UPD_EX && \
+    echo "chown -R $NB_USER /tmp/dataset/python/examples" >> $UPD_EX && \
+    echo "chown -R $NB_USER /tmp/dataset/.git" >> $UPD_EX && \
+    echo "exit 0" >> $UPD_EX
+
+RUN bash -x $UPD_EX
+
+RUN sed -i "s/jupyter lab/jupyter lab --NotebookApp.token='' --NotebookApp.password=''/" /usr/local/bin/start-notebook.sh
 
 USER $NB_UID
 
+ARG UPD_NB=/home/$NB_USER/update_examples
+
+RUN touch $UPD_NB.py && \
+    echo "#|" >> $UPD_NB.py && \
+    echo "%%bash" >> $UPD_NB.py && \
+    echo "$UPD_EX" >> $UPD_NB.py && \
+    echo "#-" >> $UPD_NB.py && \
+    py2nb $UPD_NB.py && \
```

Comment:
![image](https://user-images.githubusercontent.com/1109536/54371594-8d74c680-4671-11e9-98cd-d6bea9fcf2a4.png)
Problem above due to the conversion? 

I have problems successfully completing a few of the notebooks created. I'm not sure that this is anything to do with your changes here. I think that's down to the notebooks getting out of step with the codebase.

Reply:
I have the same issue with demo-part1 on my normal machine. So, the problem is in the file itself.

Reply:
From the message looks like the format has been corrupted during some of the edits. Agreed, separate issue to fix this.